.gitignore

```diff
@@ -2,3 +2,5 @@ target
 */target
 Cargo.lock
 .vscode
+teos.toml
+/data_dir
```

Comment:
I typically like to remove tracking of local configuration files/setups, especially since these ones would contain btc_rpc_password etc. The data_dir and teos.toml could live outside of the repo's local folder, but it's a bit of a pain to pull that up any time someone is working on the code.

Reply:
I think both of them are meant to live outside the repo folder. It's kind of the same as with `bitcoin.conf` and `.bitcoin`, you don't expect those to be found with the Bitcoin source code.

Reply:
sounds good, I'll pull them out then